torch/_tensor_str.py

```diff
@@ -364,12 +364,13 @@ def _str_intern(inp):
         suffixes.append('size=' + str(tuple(self.shape)))
         if not has_default_dtype:
             suffixes.append('dtype=' + str(self.dtype))
-        suffixes.append('quantization_scheme=' + str(self.qscheme()))
         if self.qscheme() == torch.per_tensor_affine or self.qscheme() == torch.per_tensor_symmetric:
+            suffixes.append('quantization_scheme=per_tensor')
```

Comment:
nit: do we want to name `quantization_scheme` here to something different to distinguish this from qscheme? or maybe just print `per_tensor_affine_or_symmetric` here?

Reply:
How does `quantization_mode` sound?

Reply:
Currently we have `per_channel_affine`/`per_channel_symmetric`/`per_channel_affine_float_qparams`, so seems like just using `per_tensor_affine_or_symmetric` might also be a bit confusing. 
I think we can either just call it `quantization_scheme` or completely remove this print from the qtensor.